### PR TITLE
RDM-11717: Develop Builds Dashboard

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-11717

### Change description ###

CCD Develop build dashboard

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml [hunk 1]
```diff
@@ -92,6 +92,12 @@ spec:
                     name: CDM
                     recurse: true
                     title: CDM Builds Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_.*CDM.*\/(?!.*ccd-admin-web-api).+\/develop
```
Comment (MSancaktutar) [inline k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml:97]: Well, this regex should rather pick the PR builds with specific pr numbers.
Eg. HMCTS_CDM/job/ccd-data-store-api/view/change-requests/job/PR-1260/

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
Pls see my comment.

File: k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml [hunk 2]
```diff
@@ -92,6 +92,12 @@ spec:
                     name: CDM
                     recurse: true
                     title: CDM Builds Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_.*CDM.*\/(.*ac-manage-case-assignment.*PR-180|.*ccd-admin-web.*PR-325|.*ccd-api-gateway.*PR-353|.*ccd-case-activity-api.*PR-256|.*ccd-case-print-service.*PR-260|.*ccd-data-store-api.*PR-1417|.*ccd-definition-store-api.*PR-575|.*ccd-user-profile-api.*PR-335|.*cpo-case-payment-orders-api.*PR-54|.*ccd-admin-web.*PR-325|.*ccd-api-gateway.*PR-353|.*ccd-case-activity-api.*PR-256|.*ccd-case-print-service.*PR-260)
```
Comment (MSancaktutar) [inline k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml:97]: data store is 1260

Comment (MSancaktutar) [reply k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml:97]: pls also add case man web pr 1043.

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
pls see 2 comments

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Review by hemantt — APPROVED** [review]

**Review by timja — APPROVED** [review]
🙈

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**